### ISSUE TITLE
[A11y] Adding labels for screen readers

### ISSUE THREAD

**Issue by aarongerig** [phosphor-icons/webcomponents#9]
As mentioned in [this issue](https://github.com/phosphor-icons/homepage/issues/201), phosphor allows the user/dev to add arbitrary attributes such as `aria-label` to the icons. Unfortunately, I cannot find any documented option to do that with the web components library.

Would it be possible to add that feature or is it intended to restrict web components to not have that flexibility?

**Comment by rektdeckard** [comment]
The web components in this library are real elements, so you can already add whatever attributes are necessary to the custom element itself:

```html
<ph-link role="img" aria-label="Link icon"></ph-link>
```

As far as I know, there's no reason to apply accessible labels to anything inside the Shadow DOM; labeling the wrapping custom element should be sufficient!
